### ISSUE TITLE
Multiple Directory errors

### ISSUE THREAD

**Issue by InanimateCrbnRod** [utahjohnnymontana/DVD-Rip-Prep#1]
Hello, thank you for your work. I am on Windows, and had this script working previously. When I try using the new auto smap mode I get some errors. 
![Screenshot 2024-11-27 180719](https://github.com/user-attachments/assets/82126157-1c9e-44f1-9e3c-80d41d31feb6)
![Screenshot 2024-11-27 180731](https://github.com/user-attachments/assets/a30ce82b-446f-46ff-875b-2b1221220627)

I'm guessing its trying to read or write from a linux directory that doesn't exist on my end? Thank you.

Oh! You should also know that auto SMAP is enabled by default even though the readme says it isn't.

**Comment by utahjohnnymontana** [comment]
Sorry about that.  I keep leaving my own home directory in there.  I already fixed that, so if you download again, that problem at least should be fixed.

**Comment by InanimateCrbnRod** [comment]
Thank you, that got rid of some of the error messages. Here's what I'm encountering now:
![Screenshot 2024-11-27 181755](https://github.com/user-attachments/assets/5dc11bcc-b3c8-49b7-85d5-49ebf47e74ac)
![Screenshot 2024-11-27 181844](https://github.com/user-attachments/assets/4d037e3b-a243-4958-8cbe-cbe59d7c8e92)

**Comment by utahjohnnymontana** [comment]
I added a check for the default directories.  Maybe that will do the trick.

**Comment by InanimateCrbnRod** [comment]
okey dokey, now I'm getting this:

![Screenshot 2024-11-27 184235](https://github.com/user-attachments/assets/3ca25f06-1298-4b15-be28-c42d55a97292)

![Screenshot 2024-11-27 184326](https://github.com/user-attachments/assets/7fb4a25d-baa1-4e8f-a3a4-5484d563e1a0)

**Comment by utahjohnnymontana** [comment]
Well, the first is a new one for me.  I will have to do a little research to sort that out.  I imagine the second problem is a result of the first, but I'll check that out too.  The fix won't be immediate this time.

**Comment by InanimateCrbnRod** [comment]
Okay, thanks again!

**Comment by utahjohnnymontana** [comment]
I still don't know what caused this problem, but there were some miscalculations in the time adjustment that could have resulted in incorrect negative values, so it is possible that I have fixed this incidentally.

**Comment by InanimateCrbnRod** [comment]
Hi, thank you. Unfortunately I'm still getting errors.

![Screenshot 2024-12-01 230437](https://github.com/user-attachments/assets/34a9544a-ce53-413c-ab93-9a2af94556e9)

**Comment by utahjohnnymontana** [comment]
OK, it looks like it is failing to get the frame rate for that video.  Could you run mediainfo on it and paste the output?

**Comment by InanimateCrbnRod** [comment]
Yes, sorry for the noob question but can you tell me how to run mediainfo on it? I don't understand linux cli since I'm on windows. I know I have mediainfo installed correctly in the windows subsystem for linux

**Comment by InanimateCrbnRod** [comment]
Finally had a chance to figure it out. Here's what I got. 

![Screenshot 2024-12-02 220426](https://github.com/user-attachments/assets/29b3a1ac-b0e4-43a5-a6e2-e1d550610753)

**Comment by utahjohnnymontana** [comment]
What happens if you run the following command?

echo "10 / 0.333000" |bc -l

**Comment by InanimateCrbnRod** [comment]
I probably don't understand enough about command line in general, let aloe WSL command line. I think I inputted the code wrong:

![Screenshot 2024-12-04 151450](https://github.com/user-attachments/assets/03e4b558-5436-4ae6-b8b8-15876b0f344c)

**Comment by utahjohnnymontana** [comment]
OK, that helps.  I don't know anything about WSL either, so we're working a bit blind.  The script uses the bc calculator for math.  On most linux systems, this is installed by default, but maybe not under WSL.  So, you need to install bc, which might be as simple as 'sudo apt install bc'.

**Comment by InanimateCrbnRod** [comment]
Thanks, it did install BC with that command but said it was already installed and I'm getting the same error message. is this a command that runs within the DRP script or separate? Should I be running it on the file in question or just in the folder? thank you.

**Comment by ForSerious** [comment]
Found it:
smap lines 308, 318, 324, 400, 406, 412 change csv=p=0 to default=nw=1:nk=1
That'll take out the comma at the end of the number.

Question: Is the some advantage in using count_packets instead of count_frames? Could there be any unanticipated consequences?

EDIT: Oh. It still is giving a bunch of syntax errors.

**Comment by InanimateCrbnRod** [comment]
yeah I'm still getting a bunch of syntax errors too
